keyboards/clueboard/66/rev4/chconf.h

```diff
@@ -25,9 +25,6 @@
  * @{
  */
 
-#ifndef CHCONF_H
```

Comment:
These should be put back, as its now running with no guards whatsoever. (Im hesitant to suggest pragma once as its interacting with ChibiOS and thats kinda a black box).